README.md

```diff
@@ -102,6 +102,37 @@ make behat
 make mocha
 ```
 
+* Copy or create selenium-side-runner container
+
+Copy or generated with [selenium IDE](https://www.seleniumhq.org/projects/ide/).side files in the folder docker/selenium
+
+Update the hidden file .side.env with required capabilities for your tests see official doc: [selenium-side-runner](https://www.npmjs.com/package/selenium-side-runner)
+
+Build a selenium-side-runner image:
+```
+docker build -f Dockerfile-Selenium -t your_selenium-runner_name .
+```
+
+* Run GUI selenium-side-runner tests:
+
+First install [Zalenium](https://opensource.zalando.com/zalenium/)
```

Comment:
This can be achieved more easily using Docker Compose

See 70f1e2f58e100a04bf89493bb7ddbe4f991e16f7